app/src/main/java/helium314/keyboard/keyboard/Key.java

```diff
@@ -515,8 +515,11 @@ public final boolean isShift() {
     }
 
     public final boolean isModifier() {
-        return mCode == KeyCode.SHIFT || mCode == KeyCode.SYMBOL_ALPHA || mCode == KeyCode.ALPHA || mCode == KeyCode.SYMBOL
-                || mCode == KeyCode.CTRL || mCode == KeyCode.ALT || mCode == KeyCode.FN || mCode == KeyCode.META;
+        return switch (mCode) {
```

Comment:
Is the switch to `switch` just for convenience / readability, or are there other reasons too?

Reply:
That's my view of it, I think it's more legible and it's easier to add subsequent values with the commas.

Though I realize my formatting has been a bit awkward. If cases spill into a new line, should they be indented by 8 spaces or 5?

Reply:
> If cases spill into a new line, should they be indented by 8 spaces or 5?

I don't really care about this, it's readable enough anyway.